tests/table/test_name_mapping.py

```diff
@@ -247,11 +247,6 @@ def test_mapping_lookup_by_name(table_name_mapping_nested: NameMapping) -> None:
         table_name_mapping_nested.find("boom")
 
 
-def test_invalid_mapped_field() -> None:
```

Comment:
nit: can we add a test to check for `permits an empty list of names in the default name mapping`

Reply:
Done.